test/ReflectionGeneratorTest.php

```diff
@@ -10,121 +12,47 @@
  */
 class ReflectionGeneratorTest extends \PHPUnit_Framework_TestCase
 {
-    /**
-     * @dataProvider generateInIsolationProvider
-     *
-     * @param PackageClass $package_class
-     */
-    public function testGenerateInIsolation(PackageClass $package_class)
+    public function testGenerateInIsolation()
     {
-        $short_name    = $package_class->getShortName();
-        $class         = $package_class->getName();
-        $base_dir      = __DIR__ . '/EdgeCases/';
-        $expected_file = $base_dir . $short_name . 'Interface.expected.php';
-        $actual_file   = $base_dir . 'Generated/' . $short_name . 'Interface.php';
-        $expected      = file_get_contents($expected_file);
+        $base_dir    = __DIR__ . '/Functional/src/Entity';
 
-        ReflectionGenerator::generateInIsolation($class);
+        ReflectionGenerator::generateInIsolation('Hostnet\\FunctionalFixtures\\Entity\\BaseClass');
```

Comment:
yes it will work but the namespace is a bit different for the fixtures than those of the other files. So I rather have it like this for clarification